storage/postgres/queue.go

```diff
@@ -75,7 +75,7 @@ func (t *logTreeTX) dequeueLeaf(rows *sql.Rows) (*trillian.LogLeaf, dequeuedLeaf
 	return leaf, dequeueInfo(leafIDHash, queueTimestamp), nil
 }
 
-func queueArgs(treeID int64, identityHash []byte, queueTimestamp time.Time) []interface{} {
+func queueArgs(_ int64, _ []byte, queueTimestamp time.Time) []interface{} {
```

Comment:
Better yet, delete this function since it's only used in one place.

Reply:
Done.